website/docs/r/loadbalancer_nat_pool.html.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: "azurestack"
+page_title: "Azure Resource Manager: azurestack_lb_nat_pool"
+sidebar_current: "docs-azurestack-resource-loadbalancer-nat-pool"
+description: |-
+  Create a LoadBalancer NAT Pool.
+---
+
+# azurestack_lb_nat_pool
+
+Create a LoadBalancer NAT pool.
```

Comment:
Could we change this to Manages?

Reply:
Also LoadBalancer -> Load Balancer to match the azure docs (all instances of this should be fixed)